components/src/core/components/PopOver/PopOver.vue

```diff
@@ -42,20 +46,42 @@ export default defineComponent({
       type: Boolean,
       default: false,
     },
+    persistent: {
+      type: Boolean,
+      default: false,
+    },
   },
 
   emits: ['update:show'],
   // eslint-disable-next-line
-  setup: function(props: any) {
+  setup: function (props: any) {
     const isActive = ref<boolean>(props.show);
+    const isPersistent = ref<boolean>(props.persistent);
```

Comment:
really no value assigning this to new ref. you can use `props.persistent` directly since its already reactive

Reply:
Fixed